pillar/top.sls

```diff
@@ -6,5 +6,6 @@ base:
     - ip_addrs
     - fqdn
     - kubelet
+    - availability_zone
```

Comment:
It seems this is only used in the kube minions... could we use this?

```
   'G@roles:kube-minion':
      - availability-zone
```

As a minor comment, it seems our code convention for modules is that we do not use underscores, so it would be better to use `availability-zone`.